azkaban-web-server/src/main/java/azkaban/flowtrigger/FlowTriggerService.java

```diff
@@ -97,6 +98,11 @@ public FlowTriggerService(final FlowTriggerDependencyPluginManager pluginManager
     this.flowTriggerInstanceLoader = flowTriggerInstanceLoader;
   }
 
+  public void start() throws FlowTriggerDependencyPluginException {
```

Comment:
I'm seeing this variable `List<TriggerInstance> runningTriggers` exist in this class. In the long run, AZ web server will be stateless, so state should be removed eventually. For now, It might be ok to stay with this approach. Please come up with the solution to address it.

Reply:
not sure what you mean here by stateless. let's discuss more in person.

Reply:
`stateless` means more than one AZ webservers will serve web requests, and each server should not contain states internally. Could you think of any solution?

Reply:
then how would quartz scheduler handle this case? E.x, how does quartz know which schedule is served by which web server?